pkg/cmd/pr/create/create.go

```diff
@@ -141,7 +139,7 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 	fl.StringSliceVarP(&opts.Labels, "label", "l", nil, "Add labels by `name`")
 	fl.StringSliceVarP(&opts.Projects, "project", "p", nil, "Add the pull request to projects by `name`")
 	fl.StringVarP(&opts.Milestone, "milestone", "m", "", "Add the pull request to a milestone by `name`")
-	fl.StringVarP(&opts.JSONInput, "json", "j", "", "Use JSON to populate and submit PR")
+	fl.StringVarP(&opts.RecoverFile, "recover", "e", "", "Recover input from a failed run of create")
```

Comment:
In order to free up `-e` for some future flag, do you think it would be okay to expose the flag only as `--recover` without any shorthand? The way I see it, we already print copy-pasteable instructions in the error message, so it shouldn't matter that the flag should be typed in long-form.

Reply:
makes sense.